scripts/release-llvm.py

```
@@ -0,0 +1,548 @@
+    mode_tag = "debug" if mode == "Debug" else "releasedbg"
+    suffix = ""
+    if lto:
+        suffix += "-lto"
+    if asan:
+        suffix += "-asan"
+    return f"{arch}-{platform}-{toolchain}-{mode_tag}{suffix}.tar.xz"
+
+
+ARTIFACTS = [
+    build_artifact_name(p, a, m, lto=l, asan=s)
+    for p, a, m, l, s in [
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)